stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,146 @@
+[id="proc-aap-gcp-backup-and-restore"]
```

Comment:
Thanks

Reply:
Realistically there should be two files in the PR, this procedure and the assembly that includes it, you'll need to make the changes I mentioned to the assembly before this will work.